### ISSUE TITLE
Build error: Metadata file libtgvoip.winmd could not be found

### ISSUE THREAD

**Issue by magicxor** [UnigramDev/Unigram#364]
I'm trying to follow Build instructions, but for some reason VS2017 can't build the solution. I tried different branches, different platforms with no success. Maybe I missed something?

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0006	Metadata file 'C:\Users\Nonamed\Downloads\Unigram\Unigram\x64\Debug\libtgvoip.UWP\libtgvoip.winmd' could not be found	Telegram.Api	C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram.Api\CSC	1	Active
Error	CS0006	Metadata file 'C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram.Api\bin\x64\Debug\Telegram.Api.dll' could not be found	Unigram.Core	C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram.Core\CSC	1	Active
Error	CS0006	Metadata file 'C:\Users\Nonamed\Downloads\Unigram\Unigram\x64\Debug\libtgvoip.UWP\libtgvoip.winmd' could not be found	Unigram.Tasks	C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram.Tasks\CSC	1	Active
Error	CS0006	Metadata file 'C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram.Api\bin\x64\Debug\Telegram.Api.dll' could not be found	Unigram.Tasks	C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram.Tasks\CSC	1	Active
Error	CS0006	Metadata file 'C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram.Core\bin\x64\Debug\Unigram.Core.dll' could not be found	Unigram.Tasks	C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram.Tasks\CSC	1	Active
Error	CS0006	Metadata file 'C:\Users\Nonamed\Downloads\Unigram\Unigram\x64\Debug\libtgvoip.UWP\libtgvoip.winmd' could not be found	Unigram	C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram\CSC	1	Active
Error	CS0006	Metadata file 'C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram.Api\bin\x64\Debug\Telegram.Api.dll' could not be found	Unigram	C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram\CSC	1	Active
Error	CS0006	Metadata file 'C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram.Core\bin\x64\Debug\Unigram.Core.dll' could not be found	Unigram	C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram\CSC	1	Active
Error	CS0006	Metadata file 'C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram.Tasks\bin\x64\Debug\Unigram.Tasks.winmd' could not be found	Unigram	C:\Users\Nonamed\Downloads\Unigram\Unigram\Unigram\CSC	1	Active
```

**Comment by FrayxRulez** [comment]
You have to clone the repository recursively

**Comment by magicxor** [comment]
@FrayxRulez But I did so. I also tried to download libtgvoip separately and put it into it's directory.

Finally the problem was solved by installing both 10.0.14393.795 and 10.0.15063 SDK's.
